include/bpf/bpf.h

```diff
@@ -186,6 +186,112 @@ bpf_map_lookup_elem(int fd, const void* key, void* value);
 int
 bpf_map_update_elem(int fd, const void* key, const void* value, __u64 flags);
 
+/**
+ * @brief Look up multiple entries in a BPF map in batch.
```

Comment:
I'm noticing the actual English text seems to be gratuitously different from the libbpf doxygen docs at https://github.com/libbpf/libbpf/blob/master/src/bpf.h#L191

Why the difference?

Reply:
The contents in ebpf-for-windows/include/bpf/bpf.h comments are different from the libbpf doc as pointed out. I don't know the reason for deviating. Should the _batch APIs comment follow the libbpf format?

Reply:
As discussed in today's ebpf-for-windows public office hours triage,
 - Keep the same content as in libbpf.
  - Remove exception keywords.
  - File a separate task to clean up the Doxygen file for all the APIs to match libbpf format.

Reply:
Fixed.